setup.py

```diff
@@ -308,6 +308,22 @@ def run(self):
                 sys.exit(1)
 
             try:
+                node_version = check_output(['node', '--version']).rstrip()
```

Comment:
One for `node` other for `npm`. I could just condense to a little for loop instead if you prefer.